### DISCUSSION TITLE
1.2.10 has been released!

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#8943] [Announcements]
We've released v1.2.10 tonight!

The full release notes (and a longer one at that!) can be found on the releases page at [https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.10](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.10).

That said, some notable inclusions in this one are improved key security with passwords and tools to help migrate to it for your platform of choice, the addressing of the reported issue with potential node data loss on events like power losses, and several more updates as well!

**Comment by Javvegit** [comment]
Great Job!

I see the following "Added harvester info to farmer logging" does this mean we now can see all of our harvesters logging on the farm machine? If so how do I activate in in the config?

**Comment by HotDrive69** [comment]
... and with it a lot of troubles to get the wallet to work... again... Best advice is: just keep V 1.2.7...

**Comment by edaunizeau** [comment]
Hi,

Any idea of the real benefit to migrate to this version from the 1.2.3 ?

I'm afraid to have to re sync the whole network again and I do not want to loose again days before getting my wallet back...

**Comment by HotDrive69** [comment]
No point. I've updated and to make it work again I needed to resync the
blockchain... many days lost.

A quinta, 4/11/2021, 10:06, edaunizeau ***@***.***> escreveu:

**Reply by edaunizeau** [reply]
Obviously so I stay in 1.2.3.
No need to update nor loose days again ! ;)
Thanks for your reply.
